CONTRIBUTING.md

```diff
@@ -29,9 +29,10 @@ Before you begin:
 - After that, we may have questions, check back on your PR to keep up with the conversation.
 
 ### What goes where
-- The tutorials folder is for anythink showing you how to do, setup or use something.
-- The info folder is for any random information like good anticheat, scoreboard or ChatFilter plugins.
-- The recourses folder is for images for other files, configs etc.
+- The [tutorials folder](tutorials) is for any guides on how to setup/use something.
+- The [info folder](info) is for any information on commands, features etc of plugins, server jars etc.
+- The [resources folder](resources) is for images for other files, configs etc.
+- Make sure you also put the contribution is in the correct language folder.
 
 ### Your PR is merged
```

Comment:
This doesnt make sense as a heading. Possibly overhaul the file with numbered instructions?

Reply:
what do you mean?

Reply:
wait the your pr is merged?

Reply:
Yes - The  comment was line 36 / 37.....

Reply:
it was there already but what should i change it to then?

Reply:
>  Possibly overhaul the file with numbered instructions?

Reply:
i think its fine, and idk what to change it to because i didnt even add it lol